srcpkgs/ImageMagick/template

```diff
@@ -1,10 +1,10 @@
 # Template file for 'ImageMagick'
 pkgname=ImageMagick
-version=7.1.0.41
+version=7.1.0.43
 revision=1
 _majorver=${version%.*}
 _patchver=${version##*.}
-wrksrc="${pkgname}-${_majorver}-${_patchver}"
+wrksrc="${pkgname}-${_majorver}.${_patchver}"
```

Comment:
I think this is the default wrksrc value now, can you remove this line to check that?

Reply:
You mean I should get rid off the _patchver and _majorver variables and just use version as is?

Reply:
yes, unless it's used later in the template